cmd/openshift-tests/e2e.go

```diff
@@ -84,6 +84,17 @@ var staticSuites = []*ginkgo.TestSuite{
 		Parallelism: 7,
 		TestTimeout: 20 * time.Minute,
 	},
+	{
+		Name: "openshift/jenkins-e2e",
+		Description: templates.LongDesc(`
+		Tests that exercise the OpensShift / Jenkins integrations provided by the OpenShift Jenkins image/plugins and the Pipeline Build Strategy.
+		`),
+		Matches: func(name string) bool {
+			return strings.Contains(name, "openshift pipeline")
```

Comment:
time for a [jenkins] or [pipeline] tag on the relevant extended tests?

Reply:
only if you make me